From a5e22ec74f0b29ffcfb5f906298da3a50714f90f Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Wed, 9 Nov 2016 14:42:41 -0800
Subject: [PATCH] MANTA-3011 marlin worker big theory statement points to the
 wrong directory for queries Reviewed by: Robert Mustacchi <rm@joyent.com>
 Approved by: Robert Mustacchi <rm@joyent.com>

---
 jobsupervisor/lib/worker/worker.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/jobsupervisor/lib/worker/worker.js b/jobsupervisor/lib/worker/worker.js
index e1e99f2..723f2f4 100644
--- a/jobsupervisor/lib/worker/worker.js
+++ b/jobsupervisor/lib/worker/worker.js
@@ -169,7 +169,7 @@
  *     for phases 0 ... i - 1, where i = the reducer's phase.
  *
  * The detailed list of queries used for polling can be found in
- * lib/{worker,agent}/queries.js.
+ * agent/lib/agent/queries.js and jobsupervisor/lib/worker/queries.js.
  */
 
 var mod_assert = require('assert');
-- 
2.21.0

